source/includes/api/_areas_v2.md

```diff
@@ -221,53 +293,70 @@ curl -X POST https://api.resourcewatch.org/v2/area
+
+* For the email sent to users when the Area of Interest is ready to be viewed, there should exist the following email templates on Sparkpost:
+    * `dashboard-complete-zh` (Mandarin)
+    * `dashboard-complete-pt-br` (Brazilian Portuguese)
+    * `dashboard-complete-id` (Indonesian)
+    * `dashboard-complete-fr` (French)
+    * `dashboard-complete-es-mx` (Spanish)
+    * `dashboard-complete-en` (English)
+
+* For the email sent to users when the Area of Interest is being generated, there should exist the following email templates on Sparkpost:
+    * `dashboard-pending-zh` (Mandarin)
+    * `dashboard-pending-pt-br` (Brazilian Portuguese)
+    * `dashboard-pending-id` (Indonesian)
+    * `dashboard-pending-fr` (French)
+    * `dashboard-pending-es-mx` (Spanish)
+    * `dashboard-pending-en` (English)
 
-The following parameters are provided to the email service and can be used in the construction of the email:
+In order to build your templates on Sparkpost, you can use variable interpolation to customize the emails sent taking into account the area that is being processed/has been processed. While building the `dashboard-pending-*` or `dashboard-complete-*` emails, the following variables are provided and can be used in the construction of the email body:
 
 * `id` : the ID of the area.
 * `name` : the name of the area.
 * `location` : an alias for the name of the area (contains the same as the `name` parameter).
-* `subscriptions_url` : the URL to manage the areas in the frontend (example: [https://staging.globalforestwatch.org/my-gfw](https://staging.globalforestwatch.org/my-gfw)).
-* `dashboard_link` : the link to the area dashboard (example: [https://staging.globalforestwatch.org/dashboards/aoi/5d517b3fb8cfd4001061d0b2](https://staging.globalforestwatch.org/dashboards/aoi/5d517b3fb8cfd4001061d0b2)).
-* `map_link` : the "view on map" for the area (example: [https://staging.globalforestwatch.org/map/aoi/5d517b3fb8cfd4001061d0b2](https://staging.globalforestwatch.org/map/aoi/5d517b3fb8cfd4001061d0b2)).
+* `subscriptions_url` : the URL for managing areas of interest in the flagship application (example: [https://staging.globalforestwatch.org/my-gfw](https://staging.globalforestwatch.org/my-gfw)).
+* `dashboard_link` : the URL for the area dashboard (example: [https://staging.globalforestwatch.org/dashboards/aoi/5d517b3fb8cfd4001061d0b2](https://staging.globalforestwatch.org/dashboards/aoi/5d517b3fb8cfd4001061d0b2)).
+* `map_link` : the "view on map" URL for this area (example: [https://staging.globalforestwatch.org/map/aoi/5d517b3fb8cfd4001061d0b2](https://staging.globalforestwatch.org/map/aoi/5d517b3fb8cfd4001061d0b2)).
 * `image_url` : the URL for the image associated with the area.
 * `tags` : a string containing the AOI tags, comma-separated.
 
 (`5d517b3fb8cfd4001061d0b2` is an example of an area ID).
```

Comment:
replace this with the typical `<area id>``placeholder we're using in other examples.